data/vm-console-proxy-bundle/vm-console-proxy.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    control-plane: vm-console-proxy
+  name: vm-console-proxy
+  namespace: kubevirt
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: vm-console-proxy
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - kubevirt.io
+  resources:
+  - virtualmachineinstances
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - create
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - create
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: vm-console-proxy
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: vm-console-proxy
+subjects:
+- kind: ServiceAccount
+  name: vm-console-proxy
+  namespace: kubevirt
+---
+apiVersion: v1
+data:
+  tls-profile-v1alpha1.yaml: |
+    type: Intermediate
+    intermediate: {}
+kind: ConfigMap
+metadata:
+  name: vm-console-proxy
+  namespace: kubevirt
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: vm-console-proxy-cert
+  name: vm-console-proxy
+  namespace: kubevirt
+spec:
+  ports:
+  - port: 443
+    targetPort: api
+  selector:
+    vm-console-proxy.kubevirt.io: vm-console-proxy
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: vm-console-proxy
+    vm-console-proxy.kubevirt.io: vm-console-proxy
+  name: vm-console-proxy
+  namespace: kubevirt
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      vm-console-proxy.kubevirt.io: vm-console-proxy
+  template:
+    metadata:
+      labels:
+        name: vm-console-proxy
+        vm-console-proxy.kubevirt.io: vm-console-proxy
+    spec:
+      containers:
+      - args: []
+        command:
+        - /console
+        image: quay.io/kubevirt/vm-console-proxy:latest
```

Comment:
I saw that we don't have an image tagged `latest` and only `v0.1.0`. It doesn't work:
```
╭─boukhanov@andromeda ~ 
╰─$ podman pull quay.io/kubevirt/vm-console-proxy:latest
Trying to pull quay.io/kubevirt/vm-console-proxy:latest...
Error: initializing source docker://quay.io/kubevirt/vm-console-proxy:latest: reading manifest latest in quay.io/kubevirt/vm-console-proxy: manifest unknown: manifest unknown
```